bdd/lib/utils.ts

```diff
@@ -194,10 +194,13 @@ export async function waitUntilStreamEquals(stream: Readable, expected: string,
                 response += chunk;
 
                 if (response === expected) return expected;
+                if (response.length >= expected.length) {
+                    assert.equal(response, expected);
+                }
```

Comment:
maybe I'm missing something, but isn't this redundant? we have previous check for the situation when lengths are equal, and when they are unequal, the assertion will fail.

Reply:
For now we don't see reason why it failed.